cmake/LLVMIRUtil.cmake

```diff
@@ -175,16 +175,17 @@ function(llvmir_attach_bc_target)
       continue()
     endif()
 
+    get_target_property(SOURCE_DIR ${DEPENDS_TRGT} SOURCE_DIR)
+    get_filename_component(INFILE ${IN_FILE} ABSOLUTE BASE_DIR ${SOURCE_DIR})
     get_filename_component(OUTFILE ${IN_FILE} NAME)
-    get_filename_component(INFILE ${IN_FILE} ABSOLUTE)
```

Comment:
Hi compor, I was trying to follow the original naming pattern here, so I chose to maintain both `INFILE` and `IN_FILE`. Do you think it's ok if I change all `INFILE` to `ABS_IN_FILE` as you proposed?

Reply:
Yeah, but I'd say use `ABS_IN_FILE` once it is an absolute path to a file.